packages/offix-client/src/apollo/conflicts/baseHelpers.ts

```diff
@@ -34,12 +34,10 @@ export function getBaseStateFromCache(
   mutationOptions: MutationOptions
 ): ConflictResolutionData {
   const context = mutationOptions.context;
-  const idField: string = context.idField || "id";
```

Comment:
This was the whole reason id field could be passed to the mutationOptions - we wanted to know how to read base info. If we removed that we no longer need that idFIeld parameter. I have checked that quickly so not 100% sure but it is worth to mention it.

Reply:
It is still used in the mechanism where we update operations in the queue that refer to optimistic ids to real ids. example: create an object while offline and do subsequent modifications.

It may be possible at a later time to remove it completely but not yet!